default_positions.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+from sqlalchemy import Column, Integer, String, text
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import Session
+
+from __init__ import app
+from connection_initializer import init_tcp_connection_engine
+from connection_initializer import init_unix_connection_engine
+from connection_initializer import load_secrets
+
+Base = declarative_base()
+
+
+class DefaultPosition(Base):
+    """Model of DefaultPositions for SQLAlchemy."""
+
+    __tablename__ = "default_positions"
+    __table_args__ = ({"mysql_charset": "utf8mb4"})
+    id = Column(Integer, autoincrement=True, primary_key=True)
+    organizations_id = Column(Integer, nullable=False)
+    name = Column(String, nullable=False)
+    database_name = "master_data"
+
+
+class DefaultPositionManager:
+    """DefaultPositionManager for Connect Cloud SQL."""
+
+    database_engine = None
+    database_config = {
+        "echo": True,
+        "pool_size": 5,
+        "max_overflow": 2,
+        "pool_timeout": 30,
+        "pool_recycle": 1800
+    }
+    secrets = {"CLOUD_SQL_CONNECTION_NAME": "", "DB_USER": "", "DB_PASSWORD": ""}
+
+    def __init__(self):
+        """Initialize DefaultPositionManager."""
+
+        DefaultPositionManager.secrets = load_secrets(DefaultPositionManager.secrets)
+
+        if app.config["IS_LOCAL"]:
+            self.database_engine = init_tcp_connection_engine(
+                DefaultPositionManager.secrets, DefaultPosition.database_name, DefaultPositionManager.database_config)
+        else:
+            self.database_engine = init_unix_connection_engine(
+                DefaultPositionManager.secrets, DefaultPosition.database_name, DefaultPositionManager.database_config)
+
+        Base.metadata.create_all(bind=self.database_engine)
+
+    def get_default_positions(self, id: int) -> dict:
+        """Get Default Positions.
+
+        Args:
+            id (int): ID of a registered user in organizations.
+
+        Returns:
+            Dict<Default Positions>: Registration Location.
+        """
+
+        session = Session(bind=self.database_engine)
+        default_positions_dictionary = {}
+        default_positions_statement = text(
+            "select id, organizations_id, name from default_positions where id = %s" % (id))
```

Comment:
生SQL文を書くなら、SQLの予約語がどの部分かわかるように書く。

```suggestion
            "SELECT id, organizations_id, name FROM default_positions WHERE id = %s" % (id))
```

ちなみに、この程度のSELECT文ならSQLAlchemyで [`one_or_none()`](https://docs.sqlalchemy.org/en/13/orm/query.html#sqlalchemy.orm.query.Query.one_or_none)した方が良くない？
ORM的な書き方は `audit_logger.py` の `get_logs()` を参照して。

Reply:
承知しました。
そういたします。